Answers.md

```diff
@@ -2,10 +2,20 @@
 
 1.  What is React JS and what problems does it try and solve?
 
+It is a javascript frame work and it is trying to solve the issue that state is not controllable easily with vanilla js
+
 1.  What does it mean to _think_ in react?
 
+To break stuff into componenents 
+
 1.  Briefly describe some of the differences between a Class/Stateful component and a Functional/Presentational component.
 
+A statefull components have logic but functional components just return what is requested
+
 1.  Describe state.
 
+State the a variable that can store other variables and be changed
+
 1.  Describe props.
+
+It is every thing that is passed down to a functonal component 
```

Comment:
Try to give more detailed answers as these Sprint Challenge questions are potential interview questions.